scripts/glossary.coffee

```diff
@@ -0,0 +1,26 @@
+# Description:
+#   Ask for an explanation of an abbreviation/jargon
+#
+# Depdeendencies:
+#   "js-yaml": "3.4.1"
+#
+# Commands:
+#   hubot glossary <abbreviation> - returns a defined term
+#
+# Examples:
+#   hubot glossary ATO
+
+yaml = require('js-yaml');
+
+module.exports = (robot) ->
+
+  robot.respond /glossary (\w+)/i, (msg) ->
+    robot.http("https://raw.githubusercontent.com/18F/procurement-glossary/master/abbreviations.yml")
```

Comment:
I suggest hitting the GitHub Content API and base64 decoding this, instead of hot-linking to their `raw.githubusercontent.com` URL (which they don't like). 

The API URL (no auth needed) is:

https://api.github.com/repos/18F/procurement-glossary/contents/abbreviations.yml

The `content` field is base64 encoded.